DockerfileArm64

```diff
@@ -1,29 +0,0 @@
-FROM arm64v8/ubuntu:20.04
-
-RUN set -eux; \
-	apt-get update; \
-	apt-get install -y --no-install-recommends \
-		ca-certificates \
-		libpcap0.8 \
-	; \
-	rm -rf /var/lib/apt/lists/*
-
-RUN set -eux; \
-	groupadd --gid 1000 ktranslate; \
-	useradd --home-dir /etc/ktranslate --gid ktranslate --no-create-home --uid 1000 ktranslate
-
-COPY --chown=ktranslate:ktranslate config/ /etc/ktranslate/
-
-# add backwards compatibility symlinks for folks using an snmp.yml from the older image (and "ls" to verify the symlinks are correct and working)
-RUN set -eux; ln -sv ktranslate/profiles ktranslate/mibs.db /etc/; ls -l /etc/profiles/ /etc/mibs.db/
-RUN set -eux; ln -sv /etc/ktranslate/mibs.db /etc/mib.db; ls -l /etc/mib.db/
-
-# Fix libpcap to work with how we built things.
-RUN set -eux; ln -sv /usr/lib/aarch64-linux-gnu/libpcap.so.1.9.1 /usr/lib/aarch64-linux-gnu/libpcap.so.1; ls -l /usr/lib/aarch64-linux-gnu/libpcap.so.1
```

Comment:
Would we still need this line?

Reply:
In my testing on my m1 the image built and ran fine using the multi-arch ubuntu.

Reply:
awsome.